src/Take.Blip.Builder/InputExpirationHandler.cs

```diff
@@ -85,16 +87,25 @@ public Message ValidateMessage(Message message)
                     throw new ArgumentException("Message content 'StateId' must be present", nameof(InputExpiration.StateId));
                 }
 
+                var messageMetaData = new Dictionary<string, string>()
+                    {
+                        { STATE_ID, inputExpiration.StateId },
+                        { IDENTITY, inputExpiration.Identity },
+                    };
+
+                TraceSettings traceSettings = GetTraceSettings(message);
+
+                if( traceSettings != null )
+                {
+                    messageMetaData.Concat(traceSettings.GetDictionary());
```

Comment:
This was already merged, but there is a bug here: `Concat` returns a new instance with data concatened, so `messageMetaData` here is not being changed at all....

Reply:
Just saw the other PR handling this... Sorry...